server/index.js

```diff
@@ -169,27 +141,22 @@ async function main() {
     res.sendStatus(200);
   });
 
-  // endpoint for updating status of a given request
-  app.post("/requestStatus", async (req, res) => {
-    // cant seem to be able to query successfuly for some reason even though the
-    // status in the database is being updated to whatever I'm sending to the api
-    // see docs here https://neo4j.com/docs/graphql-manual/current/ogm/examples/custom-resolvers/
-    console.log("querying by: ", req.body.requestId);
-    const { r } = await Request.find({
-      where: {
-        igoRequestId: req.body.requestId
+  // custom mutation resolver for updating the sample revisable status to false
+  app.get("/mutation/sample/:smileSampleId", async (req, res) => {
+    const response = await server.executeOperation({
```

Comment:
on the update sample mutation query:
1. validate that it's revisable (not locked)
2. If revisable, flip it into revisable=false to lock it
3. Send out Nats message

Reply:
for the NATS message: unpack from request body

Reply:
this get hander can go away completely because it's being handled by graphql